### PR TITLE
Documentation for access policies

### PR DESCRIPTION
**Pull request by zivnevo**

### CODE REVIEW

File: website/content/en/docs/concepts/policies.md [hunk 1]
```diff
@@ -1,37 +1,111 @@
 ---
-title: Policies
+title: Access Policies
 description: Controlling Service access across peers
 weight: 38
 ---
 
-<!--
-Each task should give the user
+Access policies allow users and administrators a fine-grained control over which client workload may access which service.
```
Comment (elevran) [inline website/content/en/docs/concepts/policies.md:7]:
```suggestion
Access policies allow users and administrators fine-grained control over which client workloads may access which service.
```

File: website/content/en/docs/concepts/policies.md [hunk 2]
```diff
@@ -1,37 +1,111 @@
...
+This is an important security mechanism for applying [micro-segmentation](https://en.wikipedia.org/wiki/Microsegmentation_(network_security)),
+which is a basic requirement of [zero-trust](https://en.wikipedia.org/wiki/Zero_trust_security_model) systems.
```
Comment (elevran) [inline website/content/en/docs/concepts/policies.md:9]: also mention the default deny here?
Or somewhere else.

File: website/content/en/docs/concepts/policies.md [hunk 3]
```diff
@@ -1,37 +1,111 @@
...
 
-## What is it
+Every instance of an access policy either allows or denies a given set of connections.
+This set is defined by specifying the sources and destinations of these connections.
+Sources are defined in terms of the attributes attached to the client workloads.
+Destinations are defined in terms of the attributes attached to the target services.
+Both client workloads and target services may inherit some attributes from their hosting peer.
 
-### Policy Concepts
+There are two tiers of access policies in ClusterLink. The high-priority tier is intended for cluster/Peer administrators
```
Comment (elevran) [inline website/content/en/docs/concepts/policies.md:24]: nit: is the dash between high and priority needed?

File: website/content/en/docs/concepts/policies.md [hunk 4]
```diff
@@ -1,37 +1,111 @@
...
+Every instance of an access policy either allows or denies a given set of connections.
+This set is defined by specifying the sources and destinations of these connections.
+Sources are defined in terms of the attributes attached to the client workloads.
+Destinations are defined in terms of the attributes attached to the target services.
+Both client workloads and target services may inherit some attributes from their hosting peer.
 
-### Policy Concepts
+There are two tiers of access policies in ClusterLink. The high-priority tier is intended for cluster/Peer administrators 
+to set access rules which cannot be overridden by cluster users.
+High-priority policies are controlled by the `PrivilegedAccessPolicy` CRD, and are cluster-scoped (have no namespace).
```
Comment (elevran) [inline website/content/en/docs/concepts/policies.md:26]:
```suggestion
High-priority policies are controlled by the `PrivilegedAccessPolicy` CRD, and are cluster-scoped (i.e., have no namespace).
```

File: website/content/en/docs/concepts/policies.md [hunk 5]
```diff
@@ -1,37 +1,111 @@
...
+Regular policies are intended for cluster users, and are controlled by the `AccessPolicy` CRD. Regular policies are
```
Comment (elevran) [inline website/content/en/docs/concepts/policies.md:27]:
```suggestion
Regular policies are intended for cluster users, such as application developers and owners, and are controlled by the `AccessPolicy` CRD. Regular policies are
```

File: website/content/en/docs/concepts/policies.md [hunk 6]
```diff
@@ -1,37 +1,111 @@
...
+type WorkloadSetOrSelector struct {
+	WorkloadSets []string `json:"workloadSets,omitempty"`
+	WorkloadSelector *metav1.LabelSelector `json:"workloadSelector,omitempty"`
+}
+```
+
+{{% /expand %}}
+
+The `AccessPolicySpec` defines the following fields:
+- **Action** (`string`, required): whether the policy allows or denies the specified connection. Value must be either `allow` or `deny`.
```
Comment (elevran) [inline website/content/en/docs/concepts/policies.md:87]: the services.md does not use "`string`" etc., just "string" (no back ticks).

**Review by elevran — APPROVED** [review]

File: website/content/en/docs/concepts/policies.md (hunk 3 above)
Comment (zivnevo) [reply website/content/en/docs/concepts/policies.md:26]: Yes. See [here](https://www.grammarly.com/blog/hyphen-in-compounds-with-high-low/)

**Review by elevran — APPROVED** [review]

**Pull request merged**